src/app/components/views/home/home.component.html

```diff
@@ -1 +1,18 @@
-<p>home works!</p>
+
+<div>
+    <div class="text1">
+        <h3 class="heading">Store your important urls in a smart way</h3>
+        <p class="para">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Consequatur voluptas totam quibusdam nihil! Odio distinctio cum in magni enim facere quaerat minus itaque! Ullam enim obcaecati similique laboriosam tempore beatae.</p>
```

Comment:
@arjunkarthikeyanakka do we have anything to replace the lorem ipsum with? Else we can remove the paragraphs completely.

Reply:
Maybe it can say that a user can store almost 10-15 URLs , and this website helps them to store their important URLs in a safe space . They would also have control over which URLs they want to delete or modify or copy .

Reply:
Okay sounds good.

Reply:
can any one send the text to keep their

Reply:
Let's frame the exact sentences later, I just wanted to make sure if we should have it all or not.